style.css

```diff
@@ -361,15 +367,16 @@ html {
 
 /* Section Contact Form */
 
+/* Use a class to store the display flex atribute, instead of repeating */
+
 .contactform {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: left;
-  width: auto;
+  width: 100%;
   border-radius: 5px;
   height: auto;
-  margin: 0;
   padding: 1.5rem;
   background: url("./Assets/Images/illustration-contact-form.svg");
   background-repeat: no-repeat;
```

Comment:
Kindly update the styling of your contact form to match the figma design better🙏

Your design:
<img width="148" alt="image" src="https://user-images.githubusercontent.com/37116322/221924800-86c75072-c792-407e-8e7d-bde13d3f5e6a.png">


Figma design:
<img width="127" alt="image" src="https://user-images.githubusercontent.com/37116322/221924895-0027f61a-7400-47df-8900-c49f454d4822.png">

A few things to note:
- Please update the background image of the contact section to be at the bottom not at the top
- Please add some left padding to all your form inputs.(the textarea should also have top padding)
- Please update the email address at the bottom to have the same font styling as the figma template

**Note: I had to first fix the link to your stylesheet to see the styles applied to your webpage**

Reply:
Thank you for the feedbacks, I implemented:

- Background image repositioning;
- Added padding to the form input fields;
- Updated font at the final part with the email.